piano-hero/mobile/src/main/java/com/novoda/pianohero/MidiKeyboardDriver.java

```diff
@@ -1,263 +1,51 @@
 package com.novoda.pianohero;
 
 import android.content.Context;
-import android.hardware.usb.UsbDevice;
-import android.support.annotation.NonNull;
-import android.util.Log;
-import android.util.SparseArray;
 
 import jp.kshoji.driver.midi.device.MidiInputDevice;
-import jp.kshoji.driver.midi.device.MidiOutputDevice;
-import jp.kshoji.driver.midi.util.UsbMidiDriver;
 
 interface MidiKeyboardDriver {
 
-    void attachListener(KeyListener keyListener);
+    void attachListener(NoteListener noteListener);
 
     void open();
 
     void close();
 
-    interface KeyListener {
+    class KeyStationMini32 extends SimpleUsbMidiDriver implements MidiKeyboardDriver {
+        private NoteListener noteListener;
 
-        void onKeyPressed(Note note);
-
-    }
-
-    enum Note {
```

Comment:
the enum would be helpful for human comparison - since we're sticking to the key of C, formatting midi numbers to the relevant human readable note name is ezpz (SimplePitchNotationFormatter)